### PR TITLE
Video animation in default browser interstitial

### PR DESCRIPTION
**Pull request by chung-lai**
Adding an arm with video animation

### How was this tested?
- [x] I tested this manually
- [ ] I added automated tests
- [ ] Existing automated tests are enough to cover my changes

https://user-images.githubusercontent.com/87332578/173695608-1a9ccab6-ddf8-4ab9-a5e9-1163b331cc1b.mov

| screen 2.5 | video |
| --- | --- |
| Screen Shot 2022-06-14 at 3 22 53 PM | Screen Shot 2022-06-14 at 3 22 57 PM |
| Screen Shot 2022-06-14 at 1 47 07 PM | Screen Shot 2022-06-14 at 1 47 09 PM |

### CODE REVIEW

**Comment by hwiatneeva** [comment]
from design

1. **show me** button vertical spacing:  both up and down -- double (2x) the spacing (prevent mis-tapping of CTA)
2. **video* bottom vertical spacing (between the video and CTA) -- triple (3x) the spacing (it's just too narrow)

Thx!!

File: Client/Application/SceneDelegate.swift [hunk 1]
```diff
@@ -118,6 +118,11 @@ class SceneDelegate: UIResponder, UIWindowSceneDelegate {
         getAppDelegate().setUpWebServer(getAppDelegate().profile)
 
         NotificationPermissionHelper.shared.updatePermissionState()
+
+        // Continue playing the video if the
```
Comment (jedireza) [inline Client/Application/SceneDelegate.swift:122]: Incomplete comment?

**Review by jedireza — APPROVED** [review]
Code looks good. I was going to mention the vertical spacing, but Hwi beat me to it.

**Comment by darinwf** [comment]
It does not look like this PR landed on `main`, so I'm not sure if I should cherry-pick it.

**Pull request merged**